python/README.md

```diff
@@ -17,11 +17,12 @@
 
 # Iceberg Python
 
-Iceberg is a python library for programatic access to iceberg table metadata as well as data access. The intention is to provide a functional subset of the java library.
+Apache-iceberg is a python library for programmatic access to iceberg table metadata as well as data access. 
```

Comment:
nit:  Project name should just be `Apache Iceberg` (no dash).

Reply:
Thanks for the comments and will update it and also put the actual package name as well.